examples/common/CMakeModules/CommonGenevaBuild.cmake

```diff
@@ -160,16 +160,32 @@ ENDIF ()
 
 SET (Boost_USE_MULTITHREAD ON)
 SET (Boost_ADDITIONAL_VERSIONS
-        "1.70"
-        "1.70.0"
-        "1.71"
-        "1.71.0"
-        "1.72"
-        "1.72.0"
-        "1.73"
-        "1.73.0"
-        "1.74"
-        "1.74.0"
+		"1.70"
```

Comment:
Here all lines were deleted, but still probably tabs instead of spaces? (indentation changed, i did not check)